volttron/platform/agent/base_aggregate_historian.py

```diff
@@ -290,8 +284,8 @@ def _init_agg_group(self, agg_group, agg_time_period):
             _log.debug("End of loop in init_agg_group. ids {}".format(
                 data['topic_ids']))
 
-    def _collect_aggregate_data(self, collection_time, agg_time_period,
-                                use_calendar_periods, points):
+    def collect_aggregate_data(self, collection_time, agg_time_period,
```

Comment:
Why are we changing the _ of these.  Are these meant to be called from outside this module?